po/ja.po

```diff
@@ -3758,162 +3758,214 @@ msgid ""
 #: src/pattern-matching.md:25 src/pattern-matching/match-guards.md:22
 #: src/structs.md:31 src/methods.md:30 src/methods/example.md:46
 msgid "Key Points:"
-msgstr ""
+msgstr "キーポイント: "
 
 #: src/control-flow/blocks.md:46
 msgid ""
 "The point of this slide is to show that blocks have a type and value in "
 "Rust. "
 msgstr ""
+"このスライドのポイントは、Rust における block は型と値を持つということです。"
 
 #: src/control-flow/blocks.md:47
 msgid ""
 "You can show how the value of the block changes by changing the last line in "
 "the block. For instance, adding/removing a semicolon or using a `return`."
 msgstr ""
+"ブロック内にある最後の行を変更することによって、ブロック全体の値が変わること"
+"が分かります。例えば、行末のセミコロンを追加/削除したり、`return` を使用した"
+"りすることで、ブロックの値は変化します。"
 
 #: src/control-flow/if-expressions.md:1
 msgid "`if` expressions"
-msgstr ""
+msgstr "`if` 式"
 
 #: src/control-flow/if-expressions.md:3
 msgid ""
 "You use [`if` expressions](https://doc.rust-lang.org/reference/expressions/"
 "if-expr.html#if-expressions) exactly like `if` statements in other languages:"
 msgstr ""
+"Rust の [`if` 式](https://doc.rust-lang.org/reference/expressions/if-expr."
+"html#if-expressions) は、他の言語における `if` 文と全く同じように使えます。"
 
 #: src/control-flow/if-expressions.md:18
 msgid ""
 "In addition, you can use `if` as an expression. The last expression of each "
 "block becomes the value of the `if` expression:"
 msgstr ""
+"さらに、`if` を式としても用いることができます。それぞれのブロックにある最後の"
+"式が、`if` 式の値となります。"
 
 #: src/control-flow/if-expressions.md:35
 msgid ""
 "Because `if` is an expression and must have a particular type, both of its "
 "branch blocks must have the same type. Consider showing what happens if you "
 "add `;` after `x / 2` in the second example."
 msgstr ""
+"`if` は式であるため、1 つの決まった型を持たなくてはなりません。したがって、"
+"`if` 式の分岐ブロックは同一の型を持つ必要があります。2 つ目の例において、"
+"`x / 2` のあとに `;` を付け加えると何が起こるでしょうか。"
 
 #: src/control-flow/for-expressions.md:1
 msgid "`for` loops"
-msgstr ""
+msgstr "`for` ループ"
 
 #: src/control-flow/for-expressions.md:3
 msgid ""
 "The [`for` loop](https://doc.rust-lang.org/std/keyword.for.html) is closely "
 "related to the [`while let` loop](while-let-expressions.md). It will "
 "automatically call `into_iter()` on the expression and then iterate over it:"
 msgstr ""
+"[`for` loop](https://doc.rust-lang.org/std/keyword.for.html) は、[`while "
+"let` loop](while-let-expressions.md) とよく似ています。`for` ループは `in` "
+"キーワードの右側にある式に対して自動的に `into_iter()` を呼び出し、その結果生"
+"成されたイテレータを用いて走査を行います。"
 
 #: src/control-flow/for-expressions.md:22
 msgid "You can use `break` and `continue` here as usual."
 msgstr ""
+"`for` ループの中では、いつも通り `break` や `continue` を使うことができます。"
 
 #: src/control-flow/for-expressions.md:26
 msgid "Index iteration is not a special syntax in Rust for just that case."
 msgstr ""
+"Rust では、インデックスによる反復処理のために特別な構文は提供されていません。"
 
 #: src/control-flow/for-expressions.md:27
 msgid "`(0..10)` is a range that implements an `Iterator` trait. "
-msgstr ""
+msgstr "`(0..10)` は `Iterator` トレイトを実装した range です。"
```

Comment:
Perhaps we need to create another PR to the original text, but we could add more clarity by updating "range" with "Range". By this modification, the audiences could understand that (0..10) defines a Range type data which implements Iterator trait as they can find the Range data type, not range.

Reply:
Thank you for your review. As you pointed out, "Range" is a type, so I have promptly changed the Japanese translation to "Range" and made adjustments so that it's clear it's a type.